.github/gh-actions-self-hosted-runners/arc/config/arc_deployment.tpl

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+apiVersion: actions.summerwind.dev/v1alpha1
+kind: RunnerDeployment
+metadata:
+  name: main-runners
+spec:
+  template:
+    spec:
+      image: summerwind/actions-runner:v2.304.0-ubuntu-20.04-30355f7
```

Comment:
Why did we decide on this image? Looks like this is maybe an alias for actions? Might be worth dropping in a comment

Reply:
This is the image that is used by default by the controller. I only pinned the version that was latest at the time.
Image itself looks fine to use and probably works best with the controller